ChocolateyGet.psd1

```diff
@@ -1,11 +1,11 @@
 @{
 
 RootModule = 'ChocolateyGet.psm1'
-ModuleVersion = '1.0.0'
+ModuleVersion = '1.0.1'
```

Comment:
Please update the version to 1.0.0.2.  My bad, I used 1.0.0.1 in the https://www.powershellgallery.com/packages/ChocolateyGet/1.0.0.1.

Reply:
Fixed in 10add078fd30ebef7302b279bffa2356ad47fc68